src/Calendar.xml

```diff
@@ -0,0 +1,250 @@
+        <property key="titleAttribute" type="attribute" entityProperty="eventEntity">
+            <caption>Title</caption>
+            <category>Data source</category>
+            <description/>
+            <attributeTypes>
+                <attributeType name="String"/>
+            </attributeTypes>
+        </property>
+        <property key="allDayAttribute" type="attribute" entityProperty="eventEntity" required="false">
+            <caption>All day</caption>
+            <category>Data source</category>
+            <description/>
+            <attributeTypes>
+                <attributeType name="Boolean"/>
+            </attributeTypes>
+        </property>
+        <property key="startAttribute" type="attribute" entityProperty="eventEntity" required="false">
+            <caption>Start</caption>
+            <category>Data source</category>
+            <description/>
+            <attributeTypes>
+                <attributeType name="DateTime"/>
+            </attributeTypes>
+        </property>
+        <property key="endAttribute" type="attribute" entityProperty="eventEntity" required="false">
+            <caption>End</caption>
+            <category>Data source</category>
+            <description/>
+            <attributeTypes>
+                <attributeType name="DateTime"/>
+            </attributeTypes>
+        </property>
+        <property key="eventColor" type="attribute" entityProperty="eventEntity" required="false">
+            <caption>Event color</caption>
+            <category>Data source</category>
+            <description/>
+            <attributeTypes>
+                <attributeType name="Enum" />
+                <attributeType name="String" />
+            </attributeTypes>
+        </property>
+        <property key="limitEvents" type="boolean" defaultValue="false">
+            <caption>Limit events</caption>
+            <category>Data source</category>
+            <description>Boolean attribute to define whether the amount of events for one single day should be limited</description>
+        </property>
+        <property key="firstDayOfTheWeek" type="integer" defaultValue="0">
+            <caption>First day of the week</caption>
+            <category>Data source</category>
+            <description>An integer to specify the first day of the week, 0 = Sunday, 1 = Monday etc.</description>
+        </property>
+        <property key="defaultView" type="enumeration" defaultValue="month">
+            <caption>Default view</caption>
+            <category>Appearance</category>
+            <description/>
+            <enumerationValues>
+                <enumerationValue key="month">Month</enumerationValue>
+                <enumerationValue key="week">Week</enumerationValue>
+                <enumerationValue key="work_week">Work week</enumerationValue>
+                <enumerationValue key="day">Day</enumerationValue>
+                <enumerationValue key="agenda">Agenda</enumerationValue>
```

Comment:
Not really sure what that means if there is another name for this maybe it would be better to use it